client/main.lua

```diff
@@ -1864,6 +1875,28 @@ function RefreshJobBlips()
 				AddTextComponentString(_U('blip_police_pound'))
 				EndTextCommandSetBlipName(blip)
 				table.insert(JobBlips, blip)
+				end
+			end
+		end
+	end
+
+	if Config.UseMechanicPounds and Config.UseMechanicBlips then
+		if ESX.PlayerData.job and ESX.PlayerData.job.name == 'mechanic' then
+			for k,v in pairs(Config.MechanicPounds) do
+				if not v.Hidden then
+				local blip = AddBlipForCoord(v.Marker)
+
+				SetBlipSprite (blip, Config.JPoundBlip.Sprite)
+				SetBlipColour (blip, Config.JPoundBlip.Color)
+				SetBlipDisplay(blip, Config.JPoundBlip.Display)
+				SetBlipScale  (blip, Config.JPoundBlip.Scale)
+				SetBlipAsShortRange(blip, true)
+
+				BeginTextCommandSetBlipName("STRING")
+				AddTextComponentString(_U('blip_mechanic_pound'))
```

Comment:
Missing value in locales/en.lua

Reply:
![image](https://user-images.githubusercontent.com/66789826/84567524-d43cf900-adab-11ea-83ca-ba779e909e90.png)

Yes po meron na hehe.